lib/googleapis.ts

```diff
@@ -211,4 +211,4 @@ GoogleApis.prototype.discoverAPI = function (path, options, callback) {
  * @module googleapis
  * @type {GoogleApis}
  */
-export = new GoogleApis();
+export default GoogleApis;
```

Comment:
The Google TS style guide disallows default exports. This would be semver-major, but this change is semver major either way. /cc @lukesneeringer @JustinBeckwith

Reply:
See also this comment about lack of consistency in default export usage: https://github.com/google/google-api-nodejs-client/issues/803#issuecomment-324934917. I think it is a fair criticism.

Reply:
apirequest.ts and discovery.ts use the export default syntax as well. Do we need to convert these to a class based declaration like the one used in the auth library?

Reply:
@bantini yes, it would be good to avoid default exports from those files too. However, those are not public APIs for this module, so you can fix them in a separate PR.

Reply:
What's the status of this?

Reply:
With all the new changes, it would be easier to start again rather than repair